test/Api.Test/Api.Test.csproj

```diff
@@ -1,8 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-

```

Comment:
Will the namespace change here (from undefined to `Bit.Api.Test`) impact the test run or build?

Reply:
The impact it could have is on embedded resources. The name to an embedded resource would change from `Api.Test.embedded_resource.txt` to `Bit.Api.Test.embedded_resource.txt`.  I don't see any embedded resources so I don't think this should impact the test run or build. All other projects should have the exact same `RootNamespace` so their resource names would remain the same. [Microsofts explanation of what `RootNamespace` is used for.](https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2019)

The other, not as important change, is when creating a new class file in Visual Studio. The namespace would begin with `Bit` as before it would not but I would assume that would be preferred because the two existing controller tests begin with `Bit`.